### PR TITLE
Eren development put series

### PR DESCRIPTION
**Pull request by isErenG**
I addd the put series, could you please check if it is how you wanted it. I have no idea what I was doing. I had to delete two lines from docs.go because it was incorrect fyi.

### CODE REVIEW

File: internal/delivery/http/handlers/lego/legoseries/update_series.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"net/http"
+	"strconv"
+)
+
+// UpdateSeries
+//
+//	@Summary	Update LegoSeries object
+//	@Tags		lego_series_admin
+//	@ID			put_series
+//	@Param		seriesID	path	int	true	"series ID"
```
Comment (wjojf) [inline internal/delivery/http/handlers/lego/legoseries/update_series.go:17]: No Param for request body

File: internal/delivery/http/handlers/lego/legoseries/update_series.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//
+//	@Security	JWT
+func (lsh *LegoSeriesHandler) UpdateSeries(c *gin.Context) {
+	var seriesRequest lego.LegoSeriesRequest
+	if _err := c.ShouldBindJSON(&seriesRequest); _err != nil {
+		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": _err.Error()})
+		return
+	}
+
+	setID, _err := strconv.Atoi(c.Param("seriesID"))
```
Comment (wjojf) [inline internal/delivery/http/handlers/lego/legoseries/update_series.go:31]: seriesID instead of setID

**Review by wjojf — COMMENTED** [review]
Check comments, add fixes and reopen

File: internal/delivery/http/handlers/lego/legoseries/update_series.go (hunk 1 above)
Comment (isErenG) [reply internal/delivery/http/handlers/lego/legoseries/update_series.go]: fixed

File: internal/delivery/http/handlers/lego/legoseries/update_series.go (hunk 2 above)
Comment (isErenG) [reply internal/delivery/http/handlers/lego/legoseries/update_series.go]: fixed

**Review by wjojf — APPROVED** [review]
LGTM

**Pull request merged**